adafruit_gps.py

```diff
@@ -676,6 +679,16 @@ def _parse_gsv(self, talker: bytes, data: List[str]) -> bool:
 
         return True
 
+    def _parse_pgtop(self, data) -> int:
+        # Added by Pierre Lepage.
+        if data is None or len(data) != 4:
+            return False
+
+        # Antenna status. 1: Antenna shorted. 2: Internal antenna. 3: Active (extension) antenna.
+        # User can detect change in antenna status by testing equality between item 0 and item 1 of tuple.
```

Comment:
Run `black` on this source file to reformat it. The Learn Guide discusses that: https://learn.adafruit.com/improve-your-code-with-pylint